### PR TITLE
Update Chinese Strings

### PR DESCRIPTION
**Pull request by sr093906**

### CODE REVIEW

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -1,35 +1,46 @@
 <resources>
     <string name="app_name">Blue Line Console</string>
-    <string name="loading_main_caption">正在刷新数据库......</string>
+	
+    <string name="loading_main_caption">正在刷新数据库...</string>
```
Comment (nhirokinet) [inline app/src/main/res/values-zh-rCN/strings.xml:4]: Sorry for unclear change. Can you change to &#8230; (same as English and Japanese) or keep original six dots?

English and Japanese text had three dots, and Linter claimed that three dots should be replaced with &#8230; . And checking the reason, it seemed better to follow the Linter for this. Six dots has no better alternative, so I did nothing to six dots.
https://stackoverflow.com/questions/8536375/android-replace-with-ellipsis-character

**Comment by nhirokinet** [comment]
Thank you!

**Comment by sr093906** [comment]
@nhirokinet I reverted it back. Is it possible to apply changes made to English and Japanese strings to Chinese also? Just like what happened in https://github.com/nhirokinet/bluelineconsole/compare/nhirokinet:2c9a20d...nhirokinet:8ca4478. Could you do me a favor ?

**Comment by nhirokinet** [comment]
@sr093906 If three dots (treated as one character) has no problem to you, yes, you can make pull request, or I can rewrite it in the same way if you comment so.

**Comment by sr093906** [comment]
Or how about using localization platform such as Weblate that can notify string changes and highlight changed/added ones in an effortless way. Sometimes too many 'red' and 'green' lines are not very friendly to eyesight.

**Comment by nhirokinet** [comment]
I feel your efforts like finding the string file change from commits is hard, so I will keep your suggestion in my mind.
Although for now, it isn't easy to move to localization platform now.

**Comment by nhirokinet** [comment]
I've misunderstood of 'change', I will clean up xml files placing orders and empty lines before next strings.xml change.

First I used editor included in Android Studio, and even English and Japanese has got different order. So I tried to clean up.

**Comment by nhirokinet** [comment]
And after next cleanup I will try not to move previous lines.

**Comment by sr093906** [comment]
Thanks. @nhirokinet

**Pull request merged**